src/service/poc_mobile.proto

```diff
@@ -80,6 +83,42 @@ message subscriber_location_ingest_report_v1 {
   subscriber_location_req_v1 report = 2;
 }
 
+enum signal_level {
+  NO = 0;
+  LOW = 1;
+  MEDIUM = 2;
+  HIGH = 3;
+}
+
+// Radio signal level and power in the h3 hex.
+message radio_hex_signal_level {
+  // H3 hex tile covered by the radio
+  string location = 1;
+  signal_level signal_level = 2;
+  // Signal power of the radio in dBm (decibel-milliwatts)
+  double signal_power = 3;
+}
+
+message coverage_object_req_v1 {
+  bytes pub_key = 1;
+  bytes uuid = 2;
```

Comment:
where does this `uuid` come from? how does it differ from the `pub_key` and the `cbsd_id`?